cmd/dmsg-server/commands/root.go

```diff
@@ -53,6 +59,22 @@ var rootCmd = &cobra.Command{
 		if err := sf.ParseConfig(os.Args, true, &conf); err != nil {
 			log.WithError(err).Fatal()
 		}
+		if sentryDSN != "" {
+			err := sentry.Init(sentry.ClientOptions{
+				Dsn: sentryDSN,
+				// Either set environment and release here or set the SENTRY_ENVIRONMENT
+				// and SENTRY_RELEASE environment variables.
+				Environment: "",
+				Release:     "",
+				// Enable printing of SDK debug messages.
+				// Useful when getting started or trying to figure something out.
+				Debug: true,
+			})
+			if err != nil {
+				log.Fatalf("sentry.Init: %s", err)
```

Comment:
What is expected to happen if a Sentry server is not available? Should the program restart in a loop in this case?

Reply:
I just checked this case with Sentry domain shadowed in `/etc/hosts` and everything works like no string passed at all, so everything is fine.